openslides/agenda/signals.py

```diff
@@ -15,65 +15,50 @@ def validate_start_time(value):
     try:
         datetime.strptime(value, '%d.%m.%Y %H:%M')
     except ValueError:
-        raise ValidationError(_('Invalid input.'))
+        raise DjangoValidationError(_('Invalid input.'))
```

Comment:
In this file there isn't any other ValidationError but in many other files we use the Django REST Framework ValidationError which is different from Django's ValidationError. This is just to to avoid confusion.